src/shared/store/actions/utils/errors.ts

```diff
@@ -0,0 +1,33 @@
+export class Forbidden extends Error {
+  readonly code: number = 403;
+}
+export class NotFound extends Error {
```

Comment:
Why create 1 class per error type instead of one global "RequestError" class that had a "code" property with the status code? Wouldn't it be better to have types based on a definition like: "type Forbidden is an instance of RequestError class, only if that instance has a status code of HTTP_STATUS_UNAUTHORIZED". Sorry I haven't the exact Typescript snippet here.